src/Microsoft.Identity.Web/Constants/ErrorMessage.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Web
+{
+    internal static class ErrorMessage
```

Comment:
>ErrorMessage [](start = 26, length = 12)

Would be interesting to try out adding an error identifier in the beginning of the error message, e.g. MIDW-0000001: the error text goes here. This way it will be easier to find the actual error when searching?

Reply:
Yep, Wilson does this. Is there a standard on how to properly create these error codes?

Reply:
not aware of a standard. but rules are, never re-use an id. it's ok to delete id's that are not used anymore, but it has to be ensured that id is not ever used by some other message (Wilson does this by commenting out the entry)